docs/quickstart-install.md

```diff
@@ -6,7 +6,61 @@ Install the Nillion SDK, including the <strong>`nada`</strong> tool you'll use t
 
 ## Use nilup to install the Nillion SDK
```

Comment:
can we change this title here since it's no longer nilup to something like "1-line create-nillion-app installation"

https://nillion-docs-git-feat-js-client-release-nillion.vercel.app/quickstart-install#use-nilup-to-install-the-nillion-sdk